python/cuml/cluster/dbscan.pyx

```diff
@@ -383,7 +407,7 @@ class DBSCAN(Base,
 
     @generate_docstring(skip_parameters_heading=True)
     @enable_device_interop
-    def fit(self, X, out_dtype="int32", sample_weight=None) -> "DBSCAN":
+    def fit(self, X, out_dtype="int32", sample_weight=None, method="brute_force") -> "DBSCAN":
```

Comment:
Note that [sklearn does this differently](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.DBSCAN.html#sklearn.cluster.DBSCAN): it provides an "algo" parameter for the class init function. I suggest to define a similar API.